platforms/native/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/SwiftXCTestErrorHandlingIntegrationTest.groovy

```diff
@@ -76,7 +83,10 @@ class SwiftXCTestErrorHandlingIntegrationTest extends AbstractInstalledToolChain
         def testFailure = testExecutionResult.testClass("Gradle Test Run :app:xcTest")
         testFailure.assertTestFailed(EXECUTION_FAILURE, containsText("finished with non-zero exit value"))
         if (OperatingSystem.current().isMacOsX()) {
-            testFailure.assertStderr(containsText("The bundle “AppTest.xctest” couldn’t be loaded because it is damaged or missing necessary resources"))
+            if (toolChain.version < VersionNumber.version(5, 9)) {
+                testFailure.assertStderr(containsText("The bundle “AppTest.xctest” couldn’t be loaded because it is damaged or missing necessary resources"))
+            }
+            // Else, there is no stderr/stdout produced by newer versions
```

Comment:
The stderr from `xcrun` is going missing on newer swiftc versions, in the same way it's going missing when non-ascii paths are used. I didn't get to the bottom of this, but I think it's ok for now to just make sure the test failed in the expected way.